packages/devtools/pubspec.yaml

```diff
@@ -14,6 +14,10 @@ dependencies:
   intl: ^0.15.0
   js: ^0.6.1+1
   meta: ^1.1.0
+  mutex:
+    git:
+      url: git://github.com/hoylen/dart-mutex.git
```

Comment:
is there a reason we can't use the published version of package:mutex?
https://pub.dartlang.org/packages/mutex

Reply:
No I was being overly paranoid will just use the version number.  If something happens then we can use the last good SHA.